nucleus/admin/server-mgmt/src/main/java/com/sun/enterprise/admin/servermgmt/cli/StopDomainCommand.java

```diff
@@ -77,12 +76,19 @@ public class StopDomainCommand extends LocalDomainCommand {
     Boolean force;
     @Param(optional = true, defaultValue = "false")
     Boolean kill;
-    private static final long WAIT_FOR_DAS_TIME_MS = 60000; // 1 minute
+
+    @Param(optional = true, defaultValue = "600")
```

Comment:
I wouldn't necessarily change the default from 60 seconds here.

```suggestion
    @Param(optional = true, defaultValue = "60")
```